hip/test/matrix/sellp_kernels.hip.cpp

```diff
@@ -207,7 +183,8 @@ TEST_F(Sellp, AdvancedApplyWithSliceSizeAndStrideFActorIsEquivalentToRef)
 
 TEST_F(Sellp, SimpleApplyMultipleRHSIsEquivalentToRef)
 {
-    set_up_apply_matrix();
+    set_up_apply_matrix(64);
+
```

Comment:
Same with these empty lines (and the following setup tests in this file):
```suggestion
    set_up_apply_matrix(64);
```